common/religion/holy_sites/wc_holy_sites.txt

```diff
@@ -1180,6 +1180,17 @@ taunkale = {
 #njord
 #grizzlemaw
 
+### Trogg religion
+alterac_valley = {
```

Comment:
Not localized.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/116451020-0f81d280-a86d-11eb-8fe8-2b54e2a505e1.png)

</details>